app/ui/admin.html

```diff
@@ -23,34 +23,62 @@
           ><i class="fa fa-sign-out" aria-hidden="true"></i>&nbsp; Logout</a
         >
       </div>
-      <div class="content-admin">
-        <div class="menu-list">
-          <div class="menu">
-            <div class="menu-header">
-              <h1>Bacon</h1>
-              <i
-                class="fa fa-caret-square-o-down drop-menu"
-                aria-hidden="true"
-              ></i>
-            </div>
-            <div class="menu-body">
-              <p>
-                <span class="font-bold">Description: </span>Bacon ipsum dolor
-                amet fatback pork loin strip steak burgdoggen biltong tenderloin
-                bresaola kevin rump ribeye. Frankfurter pork belly leberkas
-                shoulder, pork tongue meatloaf boudin ball tip t-bone bacon beef
-                ribs short ribs doner fatback.
-              </p>
-              <div class="menu-footer">
-                <i class="fa fa-pencil-square-o" aria-hidden="true"></i>
-                <i class="fa fa-times-circle" aria-hidden="true"></i>
-              </div>
-            </div>
-          </div>
+      <div class="background-admin">
+        <div class="add-menu">
+          <form class="admin-form" action="#" method="post">
+            <input
+              class="admin-input"
+              type="text"
+              placeholder="Enter menu name"
+            />
+            <input
+              class="admin-input"
+              type="email"
+              placeholder="Enter menu description"
+            />
+
+            <a href="#" class="btn-save-admin">Add Menu</a>
+          </form>
+        </div>
+        <div class="content-admin">
+          <div id="menu-list"></div>
+        </div>
+      </div>
+    </div>
+    <!-- Delete Modal -->
+    <div class="bg-modal-delete">
```

Comment:
<img width="593" alt="screenshot 2019-02-15 at 10 56 49 am" src="https://user-images.githubusercontent.com/24798364/52849175-b48eb580-3110-11e9-81eb-90e969e88e6d.png">
 1. The size of the modal is too big for the text displayed within it. Perhaps, reducing the size to properly accommodate the text
2. Pay attention to your text spellings
3. Properly align the text displayed within the modal. It's currently pushed too close to the top. Reducing the size of the modal and making sure that the margin at the top, bottom, left and right are equal would produce a better aesthetic feel